drivers/can/Kconfig.nxp_s32

```diff
@@ -0,0 +1,36 @@
+# Copyright 2022 NXP
+# SPDX-License-Identifier: Apache-2.0
+
+config CAN_NXP_S32_CANXL
+	bool "NXP S32 CANXL driver"
+	default y
+	depends on DT_HAS_NXP_S32_CANXL_ENABLED
+	select CAN_HAS_RX_TIMESTAMP
+	select CAN_HAS_CANFD
+	help
+	  Enable support for NXP S32 CANXL driver.
+
+if CAN_NXP_S32_CANXL
+config CAN_NXP_S32_MAX_RX
+	int "Maximum number of RX descriptors"
+	depends on CAN_NXP_S32_CANXL
+	default 16
+	range 1 128
+	help
+	  Maximum number of RX descriptors.
+
+config CAN_NXP_S32_MAX_TX
+	int "Maximum number of TX descriptors"
+	depends on CAN_NXP_S32_CANXL
+	default 16
+	range 1 128
+	help
+	  Maximum number of TX descriptors.
+
+config CAN_NXP_S32_BRS
+	bool "Frame uses CAN-FD Baud Rate Switch (BRS)"
+	depends on CAN_FD_MODE
+	default y
+	help
+	  Frame uses CAN-FD Baud Rate Switch.
```

Comment:
Other CAN drivers allow specifying BRS on a per-frame basis. Please provide more context in the help text why this is a compile-time setting for the S32 CANXL.

Reply:
NXP S32 CANXL required for set mode CAN-FD